.github/workflow-templates/browser-testing.yml

```diff
@@ -0,0 +1,63 @@
+name: Browser Testing
```

Comment:
Wanna add some comments here about what this does and what config is required to run this? Like mocha config, the npm scripts required, etc.